triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -639,16 +649,14 @@ private[lf] class Runner(
   // messages given the starting state represented by the ACS
   // argument.
   private def getTriggerEvaluator(
-      name: String,
-      acs: Seq[CreatedEvent],
+      acs: Seq[CreatedEvent]
   ): Flow[TriggerMsg, SubmitRequest, Future[SValue]] = {
-    logger.info(s"""Trigger ${name} is running as ${parties.actAs} with readAs=[${parties.readAs
-        .mkString(", ")}]""")
+    // TODO: cjp - a future PR will add in logging of trigger Id etc.
+    logger.info("Trigger starting")
```

Comment:
We no longer need to log this extra information as it is now part of the root level logging context for `Runner`